addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -9,6 +11,7 @@ const snippetsEditor = require('web_editor.snippet.editor');
 const Toolbar = require('web_editor.toolbar');
 const weWidgets = require('wysiwyg.widgets');
 
+var _t = core._t;
```

Comment:
It is use here (https://github.com/odoo-dev/odoo/blob/master-odoo-editor-integration-all/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js#L299) but was generating an error since `_t` and `Dialog` were not defined.